tests/conftest.py

```diff
@@ -144,7 +145,13 @@ def database(request):
     def drop_database():
         janitor.drop()
 
-    return "postgresql://{}@{}:{}/{}".format(pg_user, pg_host, pg_port, pg_db)
+    pass_field = ""
+    if pg_pass:
+        pass_field = f":{pg_pass}"
+
+    return "postgresql://{}{}@{}:{}/{}".format(
+        pg_user, pass_field, pg_host, pg_port, pg_db
+    )
```

Comment:
Are these changes necessary for GitHub Actions?

Reply:
Nope, good catch. That was for when we were using the `postgres:latest` container which requires a password to access the db.